pliers/filters/text.py

```diff
@@ -22,18 +22,20 @@ class TextFilter(Filter):
 
 class WordStemmingFilter(TextFilter):
 
-    ''' Nltk-based word stemming Filter.
+    ''' Nltk-based word stemming and lemmatization Filter.
 
     Args:
         stemmer (str, Stemmer): If a string, must be the name of one of the
-            stemming modules available in nltk.stem. Valid values are
-            'porter', 'snowball', 'isri', 'lancaster', 'regexp', 'wordet',
-            or 'rslp'. Alternatively, an initialized nltk StemmerI instance
+            stemming and lemmatization modules available in nltk.stem. 
+            Valid values are 'porter', 'snowball', 'isri', 'lancaster', 'regexp', 
+            'wordnet', or 'rslp'. Alternatively, an initialized nltk StemmerI instance
             can be passed.
-        tokenize (bool): if True, apply the stemmer to each token in the
-            TextStim, otherwise treat the whole TextStim as one token to stem.
+        tokenize (bool): if True, apply the stemmer/lemmatizer to each token in the
+            TextStim, otherwise treat the whole TextStim as one token to stem/lemmatize.
+        case_insensitive (bool): if True, input is lower-cased before stemming
```

Comment:
Argument name is `case_insensitive` here, but it's `case_sensitive` in the actual code.

Reply:
done